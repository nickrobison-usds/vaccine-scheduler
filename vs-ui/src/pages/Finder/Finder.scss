.finder-row {
  padding-top: 10px;
}
