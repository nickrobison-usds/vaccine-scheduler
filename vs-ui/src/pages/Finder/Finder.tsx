export const Finder: React.FC<{}> = () => {
    return (<h1>I'm the finder page</h1>)
}
