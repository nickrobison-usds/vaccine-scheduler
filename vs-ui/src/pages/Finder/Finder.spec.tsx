import {render, screen} from "@testing-library/react";

import {Finder} from "./Finder";

describe('Finder', () => {
    it('renders text', async () => {
        render(<Finder/>);
        expect(screen.getByText(/finder page/i)).toBeInTheDocument();
    });
})
