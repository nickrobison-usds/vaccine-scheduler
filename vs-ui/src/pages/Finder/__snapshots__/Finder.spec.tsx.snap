// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Finder renders text 1`] = `
<div>
  <div
    class="grid-container"
    data-testid="gridContainer"
  >
    <div
      class="grid-row finder-row"
      data-testid="grid"
    >
      <select
        class="usa-select"
        data-testid="dropdown"
        id="state"
        name="state"
      >
        <option>
           - Select -
        </option>
        <option
          value="Washington"
        >
          Washington
        </option>
        <option
          value="NJ"
        >
          New Jersey
        </option>
        <option
          value="MA"
        >
          Massachusetts
        </option>
        <option
          value="CA"
        >
          California
        </option>
        <option
          value="PA"
        >
          Pennsylvania
        </option>
        <option
          value="DC"
        >
          District of Columbia
        </option>
      </select>
    </div>
    <div
      class="grid-row finder-row"
      data-testid="grid"
    >
      <div
        class="grid-col-5"
        data-testid="grid"
      >
        <ul
          class="usa-card-group"
          data-testid="CardGroup"
        />
      </div>
      <div
        class="grid-col-7"
        data-testid="grid"
      >
        <div
          style="height: 100vh; width: 100%;"
        >
          <div
            style="width: 100%; height: 100%; margin: 0px; padding: 0px; position: relative;"
          >
            <div
              style="width: 100%; height: 100%; left: 0px; top: 0px; margin: 0px; padding: 0px; position: absolute;"
            />
            <div
              style="width: 50%; height: 50%; left: 50%; top: 50%; margin: 0px; padding: 0px; position: absolute;"
            >
              <div
                style="width: 100%; height: 100%; left: 0px; top: 0px; margin: 0px; padding: 0px; position: absolute;"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
